sopel/plugins/handlers.py

```diff
@@ -330,3 +330,51 @@ def reload(self):
         module might not be available through ``sys.path``.
         """
         self._module = self._load()
+
+
+class EntryPointPlugin(PyModulePlugin):
+    """Sopel plugin loaded from a ``setuptools`` entry point
+
+    :param entry_point: a ``setuptools`` entry point object
+
+    This handler loads a Sopel plugin exposed by a ``setuptools`` entry point
+    from the ``sopel.plugins`` group. It expects to be able to load a module
```

Comment:
> <tt>from the \`\`sopel.plugins\`\` group</tt>

The entry point plugin search function takes an optional argument to change the search group, so… should this class's docstring really talk about `sopel.plugins` as if it's the only possibility? (We won't bother changing it in this PR—I just want to start a discussion about future intentions.)

Reply:
Hm, you're right. I didn't think about that. I've an idea to fix that.